docs/http-api.md

```diff
@@ -389,16 +352,6 @@ GET /addresses/:address/parcels
 
 Returns all the parcels that belong to a given address
 
----
-
-```
-GET /parcels/:x/:y
-```
-
-### Description
-
-Returns a single parcel by its coords
-
```

Comment:
Why are we deleting the docs for this two endpoints?
Also, correct me if I'm wrong @cazala but this is now part of the docs here: https://docs.decentraland.org/decentraland/api/ so maybe we should just leave a link here:

`Moved to https://docs.decentraland.org/decentraland/api/`

Reply:
Yup, we can remove the docs from this repo and just use the one in `docs.decentraland.org`

Reply:
Sorry, should I need to remove this file?

Reply:
No, just `git pull --rebase origin master` and this diff should go away. We will remove the file and update the link in a separate PR.